### PR TITLE
Updated 0.46.0 Release note with new features

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
Updated the release note with the remaining issues in 0.46.0 release

[skip ci]
Signed-off-by: Sreekala Gopakumar sreekala.gopakumar@ibm.com

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
@TobiAjila, @pshipton - I have updated the release notes with all the remaining issues that were documented for the 0.46.0 release. Please check and merge. Thanks!

**Comment by Sreekala-Gopakumar** [comment]
@TobiAjila, @pshipton - I have updated the release notes with all the remaining issues that were documented for the 0.46.0 release. Please check and merge. Thanks!

File: doc/release-notes/0.46/0.46.md [hunk 1]
```diff
@@ -62,21 +62,59 @@ The following table covers notable changes in v0.46.0. Further information about
...
+<td valign="top">Core allocation cache statistics per thread is included in the <tt>-Xtgc:allocation</tt> report.</td>
+<td valign="top">All versions</td>
+<td valign="top">The <tt>-Xtgc:allocation</tt> option prints thread-specific allocation cache (TLH) statistics in addition to the cumulative allocation statistics.
+</td>
+</tr>
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/19323">#19323</a>, <a href="https://github.com/eclipse-openj9/openj9/pull/19442">#19442</a></td>
+<td valign="top">The <tt>-XX:[+|-]ShareOrphans</tt> option is added to control storing of orphan classes in the shared classes cache.</td>
+<td valign="top">All versions</td>
+<td valign="top">When <tt>-Xshareclasses</tt> was specified, only those class loaders that implemented the OpenJ9's public shared classes cache APIs (and its child class loaders) could store classes to the shared classes cache. Custom class loaders that did not implement these cache APIs cannot pass the module or class path information to the VM. Classes of such class loaders were not stored to the cache.
```
Comment (tajila) [inline doc/release-notes/0.46/0.46.md:98]: Minor tense changes: 

was specified, only *the* class loaders that implemented the OpenJ9's public shared classes cache APIs (and its child class loaders) could store classes to the shared classes cache. Custom class loaders that did not implement these cache APIs *couldn't* pass the module or class path information to the VM. Classes of such class loaders were not stored to the cache.

*Now, users* can enable class sharing from all class loaders, irrespective of whether the class loader implements the shared classes cache API, with the `-XX:+ShareOrphans` option.

Comment (Sreekala-Gopakumar) [reply doc/release-notes/0.46/0.46.md:98]: We avoid using the term "users". I will change the sentence to "Now, you can enable..."

**Comment by Sreekala-Gopakumar** [comment]
@tajila - I have incorporated the changes. Please check. Thanks!

**Pull request merged**